doc/nrf/external_comp/bt_fast_pair.rst

```diff
@@ -888,7 +888,29 @@ The specific use case of the Google Fast Pair application is indicated by the ch
 In the official `Fast Pair`_ documentation, the `Fast Pair Device Feature Requirements`_ category defines additional requirements for each supported use case, and specifies a list of mandatory, optional, and unsupported Fast Pair features.
 If your product is targeting one of the listed use cases, you must align your accessory firmware to meet these requirements.
 
-To learn about the software maturity levels for Google Fast Pair use cases supported by the |NCS|, see the :ref:`software_maturity_fast_pair_use_case` table.
+You can select an option depending on your use case from the following list that is part of the :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE` Kconfig choice:
+
+* :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_UNKNOWN`: the Unknown use case (the default choice) - see the :ref:`ug_bt_fast_pair_use_case_unknown` subsection for more information.
```

Comment:
```suggestion
* :kconfig:option:`CONFIG_BT_FAST_PAIR_USE_CASE_UNKNOWN` - The unknown use case (the default choice).
  See the :ref:`ug_bt_fast_pair_use_case_unknown` subsection for more information.
```
to limit the amount of different punctuation
Also, not sure if ``Unknown`` should be capitalized (the Kconfig description doesn't stick to uppercase either)